src/System.Management.Automation/FormatAndOutput/common/BaseFormattingCommand.cs

```diff
@@ -273,6 +276,23 @@ private void ProcessObject(PSObject so)
             }
         }
 
+        private void WriteNewLineObject()
+        {
+            FormatEntryData fed = new FormatEntryData();
+            fed.outOfBand = true;
```

Comment:
Isn't out of band just needed for errorrecords?

Reply:
i don't think so - it is also used for DateTime. However, it's a good point. Why is outOfBand needed here?

Reply:
A few things have it on there, but it shouldn't be added if it's not already there.

Reply:
After I remove the line I get NRE in the line:
https://github.com/PowerShell/PowerShell/blob/master/src/System.Management.Automation/FormatAndOutput/common/BaseOutputtingCommand.cs#L1061

Get-Error output:
```powershell
Type           : System.Management.Automation.CmdletInvocationException
ErrorRecord    :
    Exception             :
        Type       : System.NullReferenceException
        TargetSite :
            Name          : ProcessPayload
            DeclaringType : Microsoft.PowerShell.Commands.Internal.Format.OutCommandInner+TableOutputContext,
System.Management.Automation, Version=7.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35
            MemberType    : Method
            Module        : System.Management.Automation.dll
        StackTrace :
   at Microsoft.PowerShell.Commands.Internal.Format.OutCommandInner.TableOutputContext.ProcessPayload(FormatEntryData f
ed) in C:\Users\1\Documents\GitHub\iSazonov\PowerShell\src\System.Management.Automation\FormatAndOutput\common\BaseOutp
uttingCommand.cs:line 1059
   at Microsoft.PowerShell.Commands.Internal.Format.OutCommandInner.ProcessPayload(FormatEntryData fed, OutputContext c
) in C:\Users\1\Documents\GitHub\iSazonov\PowerShell\src\System.Management.Automation\FormatAndOutput\common\BaseOutput
tingCommand.cs:line 532
   at Microsoft.PowerShell.Commands.Internal.Format.FormatMessagesContextManager.Process(Object o) in C:\Users\1\Docume
nts\GitHub\iSazonov\PowerShell\src\System.Management.Automation\FormatAndOutput\common\FormatMsgCtxManager.cs:line 76
   at Microsoft.PowerShell.Commands.Internal.Format.OutCommandInner.ProcessObject(PSObject so) in C:\Users\1\Documents\
GitHub\iSazonov\PowerShell\src\System.Management.Automation\FormatAndOutput\common\BaseOutputtingCommand.cs:line 169
   at Microsoft.PowerShell.Commands.Internal.Format.OutCommandInner.ProcessRecord() in C:\Users\1\Documents\GitHub\iSaz
onov\PowerShell\src\System.Management.Automation\FormatAndOutput\common\BaseOutputtingCommand.cs:line 67
   at Microsoft.PowerShell.Commands.Internal.Format.FrontEndCommandBase.ProcessRecord() in C:\Users\1\Documents\GitHub\
iSazonov\PowerShell\src\System.Management.Automation\FormatAndOutput\common\BaseCommand.cs:line 178
   at System.Management.Automation.Cmdlet.DoProcessRecord() in C:\Users\1\Documents\GitHub\iSazonov\PowerShell\src\Syst
em.Management.Automation\engine\cmdlet.cs:line 173
   at System.Management.Automation.CommandProcessor.ProcessRecord() in C:\Users\1\Documents\GitHub\iSazonov\PowerShell\
src\System.Management.Automation\engine\CommandProcessor.cs:line 388
        Message    : Object reference not set to an instance of an object.
        Source     : System.Management.Automation
        HResult    : -2147467261
    CategoryInfo          : NotSpecified: (:) [out-lineoutput], NullReferenceException
    FullyQualifiedErrorId : System.NullReferenceException,Microsoft.PowerShell.Commands.OutLineOutputCommand
    InvocationInfo        :
        MyCommand       : out-lineoutput
        HistoryId       : 1
        CommandOrigin   : Internal
    ScriptStackTrace      : at <ScriptBlock>, <No file>: line 1
    PipelineIterationInfo :



TargetSite     :
    Name          : Invoke
    DeclaringType : System.Management.Automation.Runspaces.PipelineBase, System.Management.Automation, Version
=7.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35
    MemberType    : Method
    Module        : System.Management.Automation.dll
StackTrace     :
   at System.Management.Automation.Runspaces.PipelineBase.Invoke(IEnumerable input) in C:\Users\1\Documents\GitHub\iSaz
onov\PowerShell\src\System.Management.Automation\engine\hostifaces\pipelinebase.cs:line 411
   at System.Management.Automation.Runspaces.Pipeline.Invoke() in C:\Users\1\Documents\GitHub\iSazonov\PowerShell\src\S
ystem.Management.Automation\engine\hostifaces\Pipeline.cs:line 528
   at Microsoft.PowerShell.Executor.ExecuteCommandHelper(Pipeline tempPipeline, Exception& exceptionThrown, ExecutionOp
tions options) in C:\Users\1\Documents\GitHub\iSazonov\PowerShell\src\Microsoft.PowerShell.ConsoleHost\host\msh\Executo
r.cs:line 425
Message        : Object reference not set to an instance of an object.
InnerException :
    Type       : System.NullReferenceException
    TargetSite :
        Name          : ProcessPayload
        DeclaringType : Microsoft.PowerShell.Commands.Internal.Format.OutCommandInner+TableOutputContext, Syst
em.Management.Automation, Version=7.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35
        MemberType    : Method
        Module        : System.Management.Automation.dll
    StackTrace :
   at Microsoft.PowerShell.Commands.Internal.Format.OutCommandInner.TableOutputContext.ProcessPayload(FormatEntryData f
ed) in C:\Users\1\Documents\GitHub\iSazonov\PowerShell\src\System.Management.Automation\FormatAndOutput\common\BaseOutp
uttingCommand.cs:line 1059
   at Microsoft.PowerShell.Commands.Internal.Format.OutCommandInner.ProcessPayload(FormatEntryData fed, OutputContext c
) in C:\Users\1\Documents\GitHub\iSazonov\PowerShell\src\System.Management.Automation\FormatAndOutput\common\BaseOutput
tingCommand.cs:line 532
   at Microsoft.PowerShell.Commands.Internal.Format.FormatMessagesContextManager.Process(Object o) in C:\Users\1\Docume
nts\GitHub\iSazonov\PowerShell\src\System.Management.Automation\FormatAndOutput\common\FormatMsgCtxManager.cs:line 76
   at Microsoft.PowerShell.Commands.Internal.Format.OutCommandInner.ProcessObject(PSObject so) in C:\Users\1\Documents\
GitHub\iSazonov\PowerShell\src\System.Management.Automation\FormatAndOutput\common\BaseOutputtingCommand.cs:line 169
   at Microsoft.PowerShell.Commands.Internal.Format.OutCommandInner.ProcessRecord() in C:\Users\1\Documents\GitHub\iSaz
onov\PowerShell\src\System.Management.Automation\FormatAndOutput\common\BaseOutputtingCommand.cs:line 67
   at Microsoft.PowerShell.Commands.Internal.Format.FrontEndCommandBase.ProcessRecord() in C:\Users\1\Documents\GitHub\
iSazonov\PowerShell\src\System.Management.Automation\FormatAndOutput\common\BaseCommand.cs:line 178
   at System.Management.Automation.Cmdlet.DoProcessRecord() in C:\Users\1\Documents\GitHub\iSazonov\PowerShell\src\Syst
em.Management.Automation\engine\cmdlet.cs:line 173
   at System.Management.Automation.CommandProcessor.ProcessRecord() in C:\Users\1\Documents\GitHub\iSazonov\PowerShell\
src\System.Management.Automation\engine\CommandProcessor.cs:line 388
    Message    : Object reference not set to an instance of an object.
    Source     : System.Management.Automation
    HResult    : -2147467261
Source         : System.Management.Automation
HResult        : -2146233087

```

Reply:
I think we need this because we output the line _before_ any output.